vagrant/post-deploy.d/20-setup-basic-auth.sh

```diff
@@ -2,6 +2,7 @@
 set -u
 set -e
 echo "Setting up concourse basic auth as ${CONCOURSE_ATC_USER}: ${CONCOURSE_ATC_PASSWORD}"
+sed "s/--external-url.*/--external-url \'\' \\\/" -i /var/vcap/jobs/atc/bin/atc_ctl # Remove hardcoded auth redirect IP
```

Comment:
This sounds like a bug from the description in the commit message. Can you create an issue on the upstream project?